### DISCUSSION TITLE
Time to remove mozjs?

### DISCUSSION THREAD

**Discussion by jrybar-rh** [polkit-org/polkit#470] [Polls]
I'm wondering if anyone is still using this dependency.  
As per manual page, polkit has NEVER supported anything newer than ECMA Ed.5, which is the highest supported by duktape. Hence, there is no reason for dragging mozjs behind any more.  
As seen [recently](https://github.com/polkit-org/polkit/issues/12#issuecomment-2199891375), there are issues in implementation of mozjs that have already been resolved with duktape and I'm getting kind of tired of it. I think we need to move forward.  

Any objections?

**Comment by mbiebl** [comment]
Debian (co)-maintainer of polkit speaking here: The Debian package (including its derivatives like Ubuntu) is using the duktape backend. So no objections from our side if mozjs support is removed.

**Comment by Zopolis4** [comment]
We're using the duktape backend over at Chromebrew as well, so no problems here.

**Comment by Bastian-Krause** [comment]
In a Yocto project, switching polkit's PACKAGECONFIG mozjs -> duktape makes my production rootfs 64 MiB smaller.
